README.md

```diff
@@ -87,6 +92,13 @@ to the original configuration files in order to provide more consistent usage.
 - ICE_ENFORCE_LIST - choose which interfaces should be explicitly used by the gateway for the purpose of ICE candidates gathering (e.g. "eth0,192.168.0.1"), empty by default
 - ICE_IGNORE_LIST - choose which interfaces should be ignored by the gateway for the purpose of ICE candidates gathering (e.g. "eth0,192.168.0.1"), (default=vmnet)
 
+### Loaded libraries
+
+- DISABLED_PLUGINS - disables plugins, e.g. "libjanus_voicemail.so,libjanus_recordplay.so"
+- DISABLED_TRANSPORTS - disables transport libraries (default="libjanus_pfunix.so")
```

Comment:
Please make all of them empty, it was my addition to disable pfunix by default. In original janus config each default is empty and we should stick to this.